content/02.main-text.md

```diff
@@ -1,8 +1,11 @@
 ## Introduction {.page_break_before}
 
-Openness in research benefits the researchers who practice it [@doi:10.7554/eLife.16800], their scientific peers, and the public.
-Improving the availability of code and data helps ensure the methods are transparent and the results can be reproduced  [@doi:10.1038/s41562-016-0021].
-Publishing in an open access journal increases the number of readers and reviewers [@doi:10.1371/journal.pone.0000308].
+The internet enables science to be shared in real-time at a low cost to a global audience.
+This development has decreased the barriers to making science open, while supporting new massively collaborative models of research.
+However, tools must still be built whose workflows encourage openness rather than stifle it.
```

Comment:
I like the content of this sentence but the phrasing seems a little awkward.  Is it better to move a phrase and say:
> However, tools whose workflows encourage openness rather than stifle it must still be built.

or something in the active voice
> However, the scientific community requires tools whose workflows encourage openness rather than stifle it.